pkgs/development/tools/esbuild/netlify.nix

```diff
@@ -7,28 +7,16 @@
 
 buildGoModule rec {
   pname = "esbuild";
-  version = "0.13.6";
+  version = "0.14.39";
```

Comment:
Why not https://github.com/evanw/esbuild/releases/tag/v0.15.14 ?

Reply:
That's because netlify-cli requires a fork of the original esbuild with new features. The link you provided is the original esbuild codebase.